### ISSUE TITLE
Version 2.1 is not working after update weewx >3.9.2 and python3

### ISSUE THREAD

**Issue by nady2804** [gjr80/weewx-stackedwindrose#2]
Hello,
Can anybody help regarding the Windrose code?
I update my weewx to latest version 4.1.0 and change the recommended python 3 version but afterwards the windrose doesn't work!

It would be great if anybody have an idea what's going wrong and can help me.
At the moment i didn't see a chance to update the latest weewx version.

I restore my old one and run again 3.9.2

[Stackedwinrose.log](https://github.com/gjr80/weewx-stackedwindrose/files/4683641/Stackedwinrose.log)

br
Manfred

**Comment by gjr80** [comment]
Hallo Manfred,

I believe I have a WeeWX 4/python 2/3 compatible version at home. Will be back home in the next couple of days so will get you fixed up then.

Grüß
Gary

**Comment by nady2804** [comment]
Hello Gary,
Great thx very much please inform me .
Br
Manfred


Am Mi., 27. Mai 2020 um 09:01 Uhr schrieb gjr80 <notifications@github.com>:

**Comment by nady2804** [comment]
Hello Gary,
Any update on this?

Thx
Manfred

**Comment by gjr80** [comment]
Yes Manfred, just sorting out a couple of issues that have come up. Not long now.
Gary

**Comment by gjr80** [comment]
Manfred, have released v3.0.0 which is WeeWX 3.2+/4.x python 2.7+/3.x compatible. Please note there are a few changes that will require a couple of manual edits on your behalf, these are covered in the readme under Upgrading Instructions.

Please let me know if any issues.

Gary

**Comment by gjr80** [comment]
Actually you will need v3.0.1

**Comment by nady2804** [comment]
Great i will check the instructions next week
Thx

gjr80 <notifications@github.com> schrieb am So., 7. Juni 2020, 02:04:
